src/frame/sdk/UserApi.ts

```diff
@@ -25,6 +25,27 @@ export function deauthorizeDevice() {
     );
 }
 
+/**
+ * Rotate users current private key by taking their current passcode and using it to derive a key to decrypt their user private key.
+ * Then generates and augmentation factor and subtracts that augmentation factor from the users private key. The new private key is then
+ * encrypted. The new encrypted private key and the augmetnation factor is then passed as an authorized api request. The server then
+ * validates the request and augments the server side private key.
+ * @param {string} passcode Users current passcode
+ */
+export function rotateCurrUsersPrivateKey(passcode: string) {
+    const {privateKey} = ApiState.signingKeys();
```

Comment:
We're not rotating their private signing key right? I assume this should be `ApiState.encryptedUserKey()`